index.bs

```diff
@@ -168,46 +173,43 @@ Example:
 </MediaFiles>
 ```
 
-Because the creative is an HTML resource that originates in the advertiser’s
-domain, interactive component loading involves creating a cross-origin iframe.
-This means bi-directional communication between the player and creative
-requires using the postMessage API. This, in turn, means that all
-communication between player and creative is asynchronous.
-
```

Comment:
Editors note: The above paragraph felt out of place to me in this section, since it's part of the Intro section and this is fairly technical. It also felt a bit out of context. I moved it into the "Player and Creative Communication" section (below), and also made some changes within the block of copy itself.